schema.py

```diff
@@ -0,0 +1,118 @@
+from sqlalchemy import (Column, ForeignKey, DateTime, Boolean, Integer, Float, String)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy import create_engine, select
+from datetime import datetime
+import db
+
+
+"""
+This is the database for the webapp ReeDistribute
+All information collected by this program defaults to anonymous 
+
+"""
+Base = declarative_base() #Make it an OOP
+
+class IdPrimaryKeyMixin(object):
+    #An ID column we wish to add to all rows, it will be unique to the table and given incrementaly
+    id = Column(Integer, primary_key=True, nullable = False)
+
+
+class DateTimeMixin(object):
+    #To Record the time each object was created and last updated
+    created_on = Column(DateTime, default=datetime.now)

```

Comment:
the database package has datetime.now stuff built in right? (as in, do you need to get that from the stripe api or nah)